package.json

```diff
@@ -206,6 +206,10 @@
     "type": "git",
     "url": "https://github.com/wireapp/wire-webapp.git"
   },
+  "resolutions": {
+    "grunt-gitinfo/getobject": "^1.1.0",
```

Comment:
```suggestion
```

As of this PR https://github.com/wireapp/wire-webapp/pull/13841 the `grunt-gitinfo` has been removed :)